### DISCUSSION TITLE
What do I have to do to display "[" and "]".

### DISCUSSION THREAD

**Discussion by DiagProf** [spectreconsole/spectre.console#562] [Q&A] [answered]
Sorry for the stupid question. 
I would like to show "[" and "]" in the text and act stupid.

AnsiConsole.Ask<string>("Press [Enter] to navigate home.");

What is the correct escape sequence?

**Comment by patriksvensson** [comment, marked as answer]
@DiagProf You can read about escaping characters here: https://spectreconsole.net/markup

**Reply by DiagProf** [reply]
Thank you!
